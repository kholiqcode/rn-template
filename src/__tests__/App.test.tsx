import React from 'react';
import renderer from 'react-test-renderer';
import App from '../index';

describe('Testing App', () => {
  test('should render App correctly', () => {
    renderer.create(<App />);
  });
});
