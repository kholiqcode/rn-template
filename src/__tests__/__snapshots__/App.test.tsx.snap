// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing App should render App correctly 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <RNCSafeAreaProvider
    onInsetsChange={[Function]}
    style={
      Array [
        Object {
          "flex": 1,
        },
        undefined,
      ]
    }
  />
</RNCSafeAreaProvider>
`;
